src/js/spatial-navigation.js

```diff
@@ -223,7 +223,7 @@ class SpatialNavigation extends EventTarget {
    * @return {Array}
    *         Returns an array of focusable components.
    */
-  updateFocusableComponents() {
+  getFocusableComponents() {
```

Comment:
Quick question, isn't this an accidental breaking change? If so, shouldn't we remove the `npm` package of the version `8.13.0` to avoid making a breaking change?